sycl/test/xocc_tests/disabled/test_device.cpp

```diff
@@ -0,0 +1,55 @@
+// Test that CUDA code and XOCC code can cooperate in a single SYCL application.
+
+#include <CL/sycl.hpp>
```

Comment:
I think nowadays we can start using
```suggestion
#include <sycl/sycl.hpp>
```
and `sycl::` instead of `cl::sycl::`

Reply:
replacing cl::sycl:: with sycl:: still compiles but replacing CL/sycl.hpp by sycl/sycl.hpp doesn't so i left CL/sycl.hpp